src/hddsupport.c

```diff
@@ -168,12 +242,19 @@ void hddLoadModules(void)
 
         LOG("HDDSUPPORT modules loaded\n");
 
+        hddFindOPLPartition();
+
         ret = fileXioMount(hddPrefix, oplPart, FIO_MT_RDWR);
         if (ret == -ENOENT) {
             //Attempt to create the partition.
-            if ((CreateOPLPartition(oplPart, hddPrefix)) >= 0)
+            if ((hddCreateOPLPartition(oplPart)) >= 0)
                 fileXioMount(hddPrefix, oplPart, FIO_MT_RDWR);
         }
+
+        if (oplPart[5] != '+') {
+            hddCheckOPLFolder(hddPrefix);
+            gHDDPrefix = "pfs0:OPL/";
```

Comment:
Since you marked as resolved I’m guessing you saw it already but I’ll post just in case.

https://github.com/ps2homebrew/Open-PS2-Loader/blob/master/src/system.c#L932

the ‘/‘ there is the seperator after pfs0:OPL/VMC so we should be fine.